src/Store.js

```diff
@@ -148,7 +177,7 @@ Store.prototype.subscribeToState = function subscribeToState(selectors, listener
         processedListener(stateValues);
     };
     
-    for (var i = 0; i < processedSelectors.length; i++) {
```

Comment:
little confused here, definitely not at the level of JS knowledge like yourself, but does this make `i` a global? or since it was assigned at line 163 it exists here?

just realized you're not using let/const and that this project is not babelified, lol

Reply:
appears `var`'s are hoisted even inside of if blocks?